### ISSUE TITLE
Internal Server Error when editing Spain (timeout) and null wof:name

### ISSUE THREAD

**Issue by stepps00** [iandees/wof-editor#36]
https://93ctehyu4h.execute-api.us-east-1.amazonaws.com/dev/place/85633129/edit

Making edits to the record for the country of Spain results in:

>Internal Server Error
>The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application.

The size of [the file on Github](https://github.com/whosonfirst-data/whosonfirst-data-admin-es/blob/master/data/856/331/29/85633129.geojson) is less that 2mb, so I'm unsure where the problem lies. 

Javascript Console

<img width="1084" alt="Screen Shot 2020-02-06 at 12 02 32 PM" src="https://user-images.githubusercontent.com/18567700/73974092-95b4a080-48d8-11ea-9fa3-662f95c87086.png">

**Comment by iandees** [comment]
It errored trying to get the `wof:name` property. Did you delete it?

**Comment by stepps00** [comment]
I tried to delete the `wof:name` property value, yeah.. let me try again without that edit.

**Comment by stepps00** [comment]
I still hit a server error on this record, even without a `wof:name` edit...

**Comment by iandees** [comment]
I added some checks to protect against crashing when there's no `wof:name` property in daa9ff7.

My guess is that the error you saw was related to a timeout. Interacting with the GitHub API can take quite a bit of time...

**Comment by iandees** [comment]
Yea, my guess is that the validation step takes an exceptionally long time for some of these, so bumping up the timeout to 30 sec in 52861aa3509a17e16f3b270deceb9add8e37c184 allowed it to create a PR for Spain: https://github.com/whosonfirst-data/whosonfirst-data-admin-es/pull/13

**Comment by nvkelso** [comment]
Hmm. Now I can submit a PR if I set an empty `wof:name`... but that property is required and weirdly allowed to be empty. So maybe for the editor we just require its at least 1 character long (non-empty)?

https://github.com/whosonfirst-data/whosonfirst-data-admin-es/pull/14/files

**Comment by iandees** [comment]
I thought the bug was that `wof:name` should be allowed to be null (since that's what Stephen was trying to do 😄).

I can add back the check for something in `wof:name`.

**Comment by iandees** [comment]
`wof:name` is required now.

**Comment by nvkelso** [comment]
I'm still able to make PRs without name?

https://github.com/whosonfirst-data/whosonfirst-data-admin-xy/pull/14

**Comment by iandees** [comment]
Hmm, I must have been testing with a space or something? 11faed0 fixes it with a completely empty name:

![image](https://user-images.githubusercontent.com/261584/74201669-fc66f080-4c2f-11ea-9027-4a0af1d6bdfc.png)

**Comment by nvkelso** [comment]
WFM
